src/locale/i18n.ts

```diff
@@ -44,6 +45,14 @@ import {useLanguagePrefs} from '#/state/preferences'
  */
 export async function dynamicActivate(locale: AppLanguage) {
   switch (locale) {
+    case AppLanguage.an: {
+      i18n.loadAndActivate({locale, messages: messagesAn})
+      await Promise.all([
+        import('@formatjs/intl-pluralrules/locale-data/es'),
```

Comment:
```suggestion
        import('@formatjs/intl-pluralrules/locale-data/an'),
```
That's great, although I would suggest that you can leave `intl-pluralrules` as `an` because the package supports it, it's only `intl-numberformat` that doesn't :)

https://www.npmjs.com/package/@formatjs/intl-pluralrules?activeTab=code